ui: fix hardware metric labels

vladlos opened the pull request:
"Disk IOPS In Progress" => "Disk Ops In Progress", axis label => "OPS"
"Disk Read Ops" and "Disk Write Ops" => "Disk Read IOPS" and "Disk Write IOPS", axis label => "IOPS"
"Disk Read Bytes" and "Disk Write Bytes" => "Disk Read MB/s" and "Disk Write MiB/s"

Resolves: #48462

Release note (ui): update labels on hardware metrics page to be more accurate

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/57224)
<!-- Reviewable:end -->

dhartunian reviewed (commented):
@taroface can you take a quick look at these changes before we merge. They're all label changes to graphs.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-:-MNPCmtmAU9zX7po6JgE:bhxi8up)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster)

___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 64 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNPCX7l-b1flE0EkOLi:-MNPCX7l-b1flE0EkOLj:b-3339dr) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L64)):*
> ```XML
>       sources={nodeSources}
>     >
>       <Axis units={AxisUnits.Bytes} label="MB/s">
> ```

It's confusing to me that we label as "MB/s" but the field is still called `Bytes`. Can you explain this difference?


<!-- Sent from Reviewable.io -->

dhartunian commented:
@taroface friendly ping for a quick review of graph label changes here.

taroface reviewed (commented):
Sorry for the delay! I have questions about MB vs. MiB.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-:-MNyDvZX5o82w6au49T8:b-ygvbm9)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @dhartunian and @elkmaster)

___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 61 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyD9jt7HjGdQWLZGQB:-MNyD9jt7HjGdQWLZGQC:blnyxfc) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L61)):*
> ```XML
> 
>     <LineGraph
>       title="Disk Read MB/s"
> ```

Should this be `MiB/s`? I had the understanding that we were using `MiB` over `MB` on all Console metrics. Also, see below comment.
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 64 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNPCX7l-b1flE0EkOLi:-MNyCKZG0MW3n12UYcXV:b-jansjr) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L64)):*
<details><summary><i>Previously, dhartunian (David Hartunian) wrote…</i></summary><blockquote>

It's confusing to me that we label as "MB/s" but the field is still called `Bytes`. Can you explain this difference?
</blockquote></details>

Is this meant to change the y-axis of the graph? It previously read `bytes (MiB)` (the below screenshot is from my earlier v20.2.0 alpha; I assume the label was changed to `bytes` sometime afterward). Should this be `MiB/s`?

![image.png](https://files.reviewableusercontent.io/images/reviewable/8740013/DBXEUi_WnRGYyA45RxU_bYOg/image.png)
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 77 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyCx0W5TEdJn9YPADP:-MNyCx0X5iIiKyF4nuDa:bbxgkzp) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L77)):*
> ```XML
> 
>     <LineGraph
>       title="Disk Write MB/s"
> ```

Same comment as above
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 80 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyDPgUAcFAA3_cfmra:-MNyDPgUAcFAA3_cfmrb:bbxgkzp) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L80)):*
> ```XML
>       sources={nodeSources}
>     >
>       <Axis units={AxisUnits.Bytes} label="MB/s">
> ```

Same comment as above


<!-- Sent from Reviewable.io -->

dhartunian reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57224)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster)

___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 61 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyD9jt7HjGdQWLZGQB:-MO3cDEd7n1UE65041NB:be6555n) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L61)):*
<details><summary><i>Previously, taroface (Ryan Kuo) wrote…</i></summary><blockquote>

Should this be `MiB/s`? I had the understanding that we were using `MiB` over `MB` on all Console metrics. Also, see below comment.
</blockquote></details>

Correct, thanks @taroface.

@elkmaster we should use MiB instead of MB here.
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 64 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNPCX7l-b1flE0EkOLi:-MO3cGW98kWFX1szNLof:bz6b4j7) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L64)):*
<details><summary><i>Previously, taroface (Ryan Kuo) wrote…</i></summary><blockquote>

Is this meant to change the y-axis of the graph? It previously read `bytes (MiB)` (the below screenshot is from my earlier v20.2.0 alpha; I assume the label was changed to `bytes` sometime afterward). Should this be `MiB/s`?

![image.png](https://files.reviewableusercontent.io/images/reviewable/8740013/DBXEUi_WnRGYyA45RxU_bYOg/image.png)

</blockquote></details>

Thanks @taroface. These should probably stay as-is since we automatically add the (MiB) as part of the axis creation.

@elkmaster keep the Axis label as "bytes".


<!-- Sent from Reviewable.io -->

vladlos committed:
ui: fix hardware metric labels

"Disk IOPS In Progress" => "Disk Ops In Progress", axis label => "OPS"
"Disk Read Ops" and "Disk Write Ops" => "Disk Read IOPS" and "Disk Write IOPS", axis label => "IOPS"
"Disk Read Bytes" and "Disk Write Bytes" => "Disk Read MB/s" and "Disk Write MiB/s"

Resolves: #48462

Release note (ui): update labels on hardware metrics page to be more accurate

vladlos reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57224)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @dhartunian and @taroface)

___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 61 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyD9jt7HjGdQWLZGQB:-MOH6uS75WzPFdS19r_6:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L61)):*
<details><summary><i>Previously, dhartunian (David Hartunian) wrote…</i></summary><blockquote>

Correct, thanks @taroface.

@elkmaster we should use MiB instead of MB here.
</blockquote></details>

Done.
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 64 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNPCX7l-b1flE0EkOLi:-MOH6vGrDPlpH_5VM0NH:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L64)):*
<details><summary><i>Previously, dhartunian (David Hartunian) wrote…</i></summary><blockquote>

Thanks @taroface. These should probably stay as-is since we automatically add the (MiB) as part of the axis creation.

@elkmaster keep the Axis label as "bytes".
</blockquote></details>

Done.
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 77 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyCx0W5TEdJn9YPADP:-MOH6vma0mBCmKEgUZ3e:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L77)):*
<details><summary><i>Previously, taroface (Ryan Kuo) wrote…</i></summary><blockquote>

Same comment as above
</blockquote></details>

Done.
___
*[pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx, line 80 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-MNyDPgUAcFAA3_cfmra:-MOH6w56BcQDje2VH-wH:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/cd560e5cfec3b2b9be3820d855d05592934783ba/pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx#L80)):*
<details><summary><i>Previously, taroface (Ryan Kuo) wrote…</i></summary><blockquote>

Same comment as above
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

dhartunian reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57224#-:-MOcYIufDza3gUDopS4C:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @taroface)



<!-- Sent from Reviewable.io -->

dhartunian commented:
bors r+
